app/helpers/layout_helper.rb

```diff
@@ -5,7 +5,8 @@ def title(page_title, page_header = nil)
   end
 
   def title_actions(*elements)
-    content_for(:title_actions) { elements.join(" ").html_safe }
+    plugins_html = Foreman::Plugin.call_title_actions_for_path(self, request.path)
```

Comment:
You probably want to use controller + action instead of path, since that won't be useful for a #show route?